core-services/release-controller/_releases/release-okd-4.12.json

```diff
@@ -11,6 +11,18 @@
       "tagRef": {
         "name": "4.12"
       }
+    },
+    "mirror-to-okd-scos": {
+      "imageStreamRef": {
+        "namespace": "origin",
+        "name": "scos-4.12",
+        "excludeTags": [
+          "machine-config-operator",
+          "installer",
+          "installer-artifacts",
+          "fedora-coreos"
+        ]
+      }
```

Comment:
@aleskandro @bradmwilliams thinking about this again - this would effectively make OKD/SCOS a downstream of OKD/FCOS, which might have implications when OKD/FCOS fails to publish for any reason.

Couldn't we instead add this next to `mirror-to-origin` in [release-ocp-4.12-ci.json](https://github.com/openshift/release/blob/afdb7a9db76233fc270aae5ad8a351a73896ee6a/core-services/release-controller/_releases/release-ocp-4.12-ci.json#L10)? (with an adapted list of `excludeTags` for mirroring out of `ocp`).
That'd put OKD/SCOS on the same downstream level from OCP CI as OKD/FCOS.

Reply:
@LorbusChris That is how I interperated how this was implemented:
CI (OCP) -> OKD (FCOS) -> OKD (SCOS)

It sounds like what you really wanted was OCP (CI) feeding **both**, which would be done by adding the `mirror-to-okd-scos` as a peer to:  `mirror-to-origin` in `release-ocp-4.12-ci.json`

Reply:
Where (what imagestream) are you planning on injecting your OKD(SCOS) CI builds?

Reply:
The reason I ask is that any change to the OKD(SCOS) imagestream will cause a new release to be generated, so your promotions plus the OCP(CI) mirroring could potentially cause quite a bit of churning of releases

Reply:
I did the mirroring from okd/fcos to okd/scos to avoid doing new prow configs for all the other payload components...  the difference between scos and fcos is a subset of the diff between fcos/scos and ocp. But yes, this means that the mirroring of the images would only happen after an FCOS build is [accepted](https://github.com/openshift/release-controller/blob/master/cmd/release-controller/sync.go#L556) and it may be better not to depend on the fcos builds acceptance. Should I change?

Reply:
The churn should be the same as what we already have in OKD/FCOS.
We'd directly inject/promote the following images to the SCOS imagestream:
- cluster-update-keys
- cluster-samples-operator
- installer
- installer-artifacts
- machine-config-operator
- operator-marketplace
- ironic-* images